### PR TITLE
add the calssification catagory of test case (farEdge Telco NonTelco)

### PR DESCRIPTION
**Pull request by aabughosh**

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]

File: cnf-certification-test/identifiers/identifiers.go [hunk 1]
```diff
@@ -173,6 +173,12 @@ The label value is not important, only its presence.`,
 		NoDocumentedProcess,
 		bestPracticeDocV1dot3URL+" Section 5.2",
 		true,
+		claim.CategoryClassification{
+			FarEdge:  "Mandatory",
```
Comment (edcdavid) [inline cnf-certification-test/identifiers/identifiers.go:177]: should'n claim.CategoryClassification just be a map[string] string like:
type CategoryClassification struct {
   map[string] string
}
also, we could define FarEdge, Telco, NonTelco, Extended (if using a map) as constants, same with Optional, Mandatory

Comment (sebrandon1) [reply cnf-certification-test/identifiers/identifiers.go:177]: Not a bad idea, gives a lot of flexibility if we need to add additional scenarios in the future.

File: cnf-certification-test/preflight/suite.go [hunk 2]
```diff
@@ -154,7 +159,12 @@ func generatePreflightOperatorGinkgoTest(testName, description, suggestion strin
 	// Brute force-ish type of method.
 
 	// Store the test names into the Catalog map for results to be dynamically printed
-	aID := identifiers.AddCatalogEntry(testName, common.PreflightTestKey, description, suggestion, "", "", "", false, identifiers.TagCommon)
+	aID := identifiers.AddCatalogEntry(testName, common.PreflightTestKey, description, suggestion, "", "", "", false, map[string]string{
+		"FarEdge":  "Optional",
```
Comment (edcdavid) [inline cnf-certification-test/preflight/suite.go:163]: add constants for all strings in identifiers.go

**Review by edcdavid — APPROVED** [review]

**Comment by ramperher** [comment]
DCI job is failing because this change needs to be rebased, that's why we have ERROR.

**Pull request merged**